[docs] Update reporting docs (#739)
DataDog/puppet-datadog-agent: README.md

    }
    ```

>     The dogapi gem will be automatically installed. Set `manage_dogapi_gem` to false if you want to customize the installation.

Reviewer: ```suggestion
    The dogapi gem is automatically installed. Set `manage_dogapi_gem` to false if you want to customize the installation.
```
